core/app/models/spree/gallery/base.rb

```diff
@@ -0,0 +1,23 @@
+module Spree
```

Comment:
I'm not sure there's a need for this base. It should be sufficient to duck-type have the two gallery classes "quack" similarly :duck: :duck: :duck: :duck: :duck:

Reply:
:duck: 

I guess the reason I would want the base class is so that it's obvious the interface that should be implemented for galleries in general. But I understand your point and can make the changes.